From 72f64225ad443ea031616040e4227a2d42010611 Mon Sep 17 00:00:00 2001
From: Asaf Gery <asaf.gery@gmail.com>
Date: Wed, 21 Feb 2024 00:17:13 +0200
Subject: [PATCH] scripts/dtc: Remove redundant YYLOC global declaration

---
 scripts/dtc/dtc-lexer.l | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/dtc/dtc-lexer.l b/scripts/dtc/dtc-lexer.l
index 06c040902..d1b381015 100644
--- a/scripts/dtc/dtc-lexer.l
+++ b/scripts/dtc/dtc-lexer.l
@@ -38,7 +38,6 @@ LINECOMMENT	"//".*\n
 #include "srcpos.h"
 #include "dtc-parser.tab.h"
 
-YYLTYPE yylloc;
 extern bool treesource_error;
 
 /* CAUTION: this will stop working if we ever use yyless() or yyunput() */
-- 
2.43.0

